### PR TITLE
Release ggplot2 3.4.1

### LINKED ISSUE #5149: Release ggplot2 3.4.1
Prepare for release:

* [x] `git pull`
* [x] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_ggplot2.html)
* [x] [Polish NEWS](https://style.tidyverse.org/news.html#news-release)
* [x] `devtools::build_readme()`
* [x] `urlchecker::url_check()`
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `rhub::check_for_cran()`
* [x] `revdepcheck::cloud_check()`
* [x] Update `cran-comments.md`
* [x] `git push`

Submit to CRAN:

* [x] `usethis::use_version('patch')`
* [x] `devtools::submit_cran()`
* [x] Approve email

Wait for CRAN...

* [x] Accepted :tada:
* [x] `git push`
* [x] `usethis::use_github_release()`
* [x] `usethis::use_dev_version()`
* [x] `git push`

### PR DESCRIPTION
**Pull request by thomasp85**
Fix #5149

### CODE REVIEW

**Comment by teunbrand** [comment]
Would you like me to resolve the NEWS.md conflict by harmonising the news? I'm not sure what the implications of that are if we're releasing from the RC branch, i.e. does the future CRAN version get an awkward news section?

**Comment by thomasp85** [comment]
No, we shouldn't resolve anything at this point. We do that after release before merging in the rc branch.

If we did it now the news in the release would not reflect the actual release

File: DESCRIPTION [hunk 1]
```diff
@@ -2,12 +2,12 @@ Package: ggplot2
...
 Title: Create Elegant Data Visualisations Using the Grammar of Graphics
 Authors@R: c(
-    person("Hadley", "Wickham", , "hadley@rstudio.com", role = "aut",
+    person("Hadley", "Wickham", , "hadley@posit.co", role = "aut",
            comment = c(ORCID = "0000-0003-4757-117X")),
     person("Winston", "Chang", role = "aut",
            comment = c(ORCID = "0000-0002-1576-2126")),
     person("Lionel", "Henry", role = "aut"),
-    person("Thomas Lin", "Pedersen", , "thomas.pedersen@rstudio.com", role = c("aut", "cre"),
+    person("Thomas Lin", "Pedersen", , "thomas.pedersen@posit.co", role = c("aut", "cre"),
```
Comment (hadley) [inline DESCRIPTION:10]: Also need to update the RStudio line at the bottom of the authors to "Posit Software, PBC"

**Comment by yutannihilation** [comment]
@thomasp85 
Thanks for handling the release process. Now that v3.4.1 is released on CRAN, can we merge this?

**Comment by thomasp85** [comment]
Yes, I'm back from vacation next week so will do it then

**Comment by yutannihilation** [comment]
I see, I'll wait for it, thanks.

File: DESCRIPTION (hunk 1 above)
Comment (thomasp85) [reply DESCRIPTION:10]: We will have to wait for spring cleaning with this as it was already submitted

**Pull request merged**